docs/source/styles.rst

```diff
@@ -0,0 +1,52 @@
+Styles
+------
+
+SectionDoc comes with a number of predefined rendering styles
```

Comment:
You are right, my initial effort was to add the new styles in this PR, but I notices that the PR was getting too big. I should probably add a note about future work